packages/paste-core/components/badge/src/index.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+
+import {getVariantStyles, getComputedTokenNames} from './utils';
+import type {BadgeProps} from './types';
+
+export const Badge = React.forwardRef<HTMLSpanElement, BadgeProps>(
+  ({variant = 'default', children, size, ...props}, ref) => {
+    const {color, backgroundColor} = getVariantStyles(variant);
+
+    const computedTokens = size ? getComputedTokenNames(size) : {};
+
+    return (
+      <Box
+        ref={ref}
+        {...safelySpreadBoxProps(props)}
+        as="span"
+        backgroundColor={backgroundColor}
+        borderRadius="borderRadius10"
+        color={color}
+        display="inline-flex"
```

Comment:
Why not `display: inline(-block)`?

Reply:
Oh I apologize, I just saw this! 😓 The containing `span` is `inline-flex` because the inner contents are using `justify-contents` and other flexy things.